src/ucp/core/ucp_context.h

```diff
@@ -260,6 +260,22 @@ typedef struct ucp_tl_iface_atomic_flags {
     };
 
 
+/**
+ * Check if at least one feature flag from @a _flags is initialized.
+ */
+#define UCP_CONTEXT_CHECK_FEATURE_FLAGS(_context, _flags, _status, _action) \
+    do { \
+        if (ENABLE_PARAMS_CHECK && !((_context)->config.features & (_flags))) { \
+            char feature_list_str[512] = {0}; \
```

Comment:
let's use ucs_alloca(), to avoid increasing function stack size. and also ucs_unlikely